.github/workflows/deploy-to-development.yml

```diff
@@ -0,0 +1,78 @@
+on:
+  push:
+    branches-ignore:
+      - main
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  id-token: write
+
+env:
+  AWS_REGION: eu-west-2
+  IAM_ROLE_ARN: arn:aws:iam::844815912454:role/GithubActions-Terraform-Role
+  PYTHON_VERSION: 3
+  TERRAFORM_DOCS_VERSION: 0.20.0
+  TERRAFORM_VERSION: 1.11.0
+  TERRAGRUNT_VERSION: 0.78.1
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/cache@v4
+        with:
+          path: ~/.cache/pre-commit
+          key: ${{ runner.os }}-pre-commit-${{ hashFiles('.pre-commit-config.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-pre-commit-
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+
+      - uses: trade-tariff/trade-tariff-tools/.github/actions/setup-tflint@main
+
+      - run: GOBIN=/usr/local/bin/ go install github.com/terraform-docs/terraform-docs@v0.20.0
```

Comment:
why not use the official action? we could  have this run in PRs all the time

https://github.com/terraform-docs/gh-actions

Reply:
Not a terrible shout - essentially I wanted to just enable pre-commit hooks to run

Reply:
This approach just unblocked that